samples/app-sso/csharp/app_sso/appsettings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "MicrosoftAppId": "<<YOUR-MICROSOFT-APP-ID>>",
+  "MicrosoftAppPassword": "<<YOUR-MICROSOFT-APP-PASSWORD>>",
+  "ConnectionName": "<<YOUR-CONNECTION-NAME>>",
+  "SiteUrl": "<<DOMAIN-NAME>>",
+  "AzureAd": {
+    "Instance": "https://login.microsoftonline.com/",
+    "TenantId": "<<YOUR-MICROSOFT-APP-TENANT-ID>>",
+    "ClientId": "<<YOUR-MICROSOFT-APP-ID>>",
```

Comment:
There is no need to specify two time client ID and secret, maintain it in one config key and use it everywhere

Reply:
![image](https://user-images.githubusercontent.com/85157377/123813604-ad982480-d912-11eb-9ed9-e0bc168fd7c7.png)

To get Azured id options we need to add these fields are mandatory